sbt/src/sbt-test/project/unified/build.sbt

```diff
@@ -0,0 +1,12 @@
+lazy val root = (project in file("."))
+  .settings(
+    Global / cancelable := true,
+    ThisBuild / scalaVersion := "2.12.2",
```

Comment:
You mean sbt 2.0? I think we should remove it in 1.0. My rationale is the following:
  
* If we don't remove it in sbt 1.0, we cannot remove it in sbt 1.0.x. Otherwise we break compat.
* We can provide out-of-the-box syntactic rewrites so that users don't worry about this. It's just a matter of having good docs.